cli/cli_fileshare.go

```diff
@@ -109,7 +110,12 @@ func statusLoop(fileshareClient pb.FileshareClient, client transferStatusClient,
 // IsFileshareDaemonReachable returns error if fileshare daemon is not reachable, daemon not running
 // being the most likely cause
 func (c *cmd) IsFileshareDaemonReachable(ctx *cli.Context) error {
-	_, err := c.fileshareClient.Ping(context.Background(), &pb.Empty{})
+	resp, err := c.client.IsLoggedIn(context.Background(), &dpb.Empty{})
+	if err != nil || !resp.GetValue() {
```

Comment:
Why `OR !resp.GetValue()` part condition is used, in what situation it can happen?

Reply:
Actually the response is a `*pb.Bool`, so `GetValue` returns a bool where true = user is logged in, false = user is not logged in.